reference/autoscaling.html.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Autoscaling
+layout: docs
+sitemap: false
+nav: firecracker
+---
+
+Autoscaling is used to increase the number of Fly Machines based on demand. There are two forms of autoscaling supported:
+
+1. Pre-allocated Machine autoscaling
+2. Metrics-based autoscaling
+
+## Pre-allocated Machine autoscaling
```

Comment:
This title is okay...? We usually just call it "auto start and stop", which is also kind of an awkward feature name. 

What about "Traffic-based Machine shutdown and startup"? Or something similar?

Reply:
i think the places where we describe it as `auto start and stop`, it sounds fine because its context of something (i.e configuration options, etc). I think the title as is is good enough, though i don't like it very much